### DISCUSSION TITLE
[TCotD2] No more changes on 20:00 MET

### DISCUSSION THREAD

**Discussion by Tormentor667** [Realm667/Re-Releases#284] [Announcements]
If it's possible for you guys @nashmuhandes @Ozymandias81 @Blue-Shadow I would like to finish up the release package this evening as I won't be at home tomorrow, so the release on Realm667 and ModDB would be scheduled.

Is that okay for you?

**Comment by Blue-Shadow** [comment]
I'm cool with that.

**Comment by nashmuhandes** [comment]
Holy crap, so many changes literally hours before release, I _hope_ nothing broke. Looks like I have no other choice but to replay the whole thing to absolutely make sure nothing broke...

**Comment by nashmuhandes** [comment]
The secret count is now broken @Tormentor667 https://github.com/Realm667/Re-Releases/issues/285
